app/views/groups/show.html.erb

```diff
@@ -0,0 +1,34 @@
+<%= render "shared/headercreate", back: "back.png", title: "ALL TRANSACTIONS", image: "search.svg" %>
+<%= render "shared/msg" %>
+<div class="group-wrapper">
+  <% if @entities.empty? %>
+    <p class="no-trans">No transactions for this category yet</p>
+  <% else %>
+    <span class="group-amount ent-amount">
+      <%= "Total amount spent $#{@entities.sum(:amount)}" %>
+    </span>
+    <% @entities.each do |entity|%>
+      <%= link_to(entity_path(entity)) do %>
+        <ul class="group-container">
+          <li>
+            <figure>
+              <img src="<%= entity.groupe.icon %>" alt="icon">
+              <figcaption>
+                <h3><%= entity.name %></h3>
+                <small><%= entity.created_at.to_date.to_s %></small>
+              </figcaption>
+            </figure>
+            <h4 class="group-amount">
+              <%= "$#{entity.amount}" %>
+            </h4>
+          </li>
+        </ul>
+      <% end %>
+    <% end %>
+  <% end %>
+  <div class="a-link">
+    <%= link_to(new_entity_path) do %>
+      <button type="button">Add Transaction</button>
+    <% end %>
+  </div>
+</div>
```

Comment:
- [ ] I would recommend implementing the logout feature so that a user can log out into the system whenever he or she wants it.

![image](https://user-images.githubusercontent.com/55635977/166099921-363d0541-64d6-488d-b906-369082cdb778.png)

Reply:
Hello @Emmyn5600 . Thank you for this feedback. I implemented the logout option. It is the mobile nav